Makefile

```diff
@@ -215,76 +217,85 @@ install:
 	  else $(MKDIR) $(INSTALL_COMPLIBDIR); fi
 	if test -d $(INSTALL_MANDIR)/man$(MANEXT); then : ; \
 	  else $(MKDIR) $(INSTALL_MANDIR)/man$(MANEXT); fi
-	cp VERSION $(INSTALL_LIBDIR)/
-	cd $(INSTALL_LIBDIR); rm -f \
+	cp -- VERSION $(INSTALL_LIBDIR)/
```

Comment:
Interestingly enough, the `--` option is not documented for `cp` or `cd`. Have you checked that it's portable across all our supported architectures? (Think of the BSDs!)

Moreover, this change will be superseded by [MPR#5219](http://caml.inria.fr/mantis/view.php?id=5219) at some point.